### PR TITLE
Update the `<ConfirmBid>` component to show a spinner while loading

### PR DESCRIPTION
**Pull request by yuki24**
finishes https://artsyproduct.atlassian.net/browse/PURCHASE-131

 * [x] ~Send a separate PR to update `src/lib/Components/Buttons/InvertedButton.tsx`~ done: https://github.com/artsy/emission/pull/1053
 * [ ] Update tests

## Screenshot

out

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  [danger-plugin-jest] Could not read test results. Danger cannot pass or fail the build.
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Components/Bidding/Components/Button.tsx [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+import InvertedButton from "lib/Components/Buttons/InvertedButton"
+import { ButtonProps, GhostButton } from "../../Buttons"
 
 export class Button extends React.Component<ButtonProps> {
   render() {
     const { style, textStyle, ...props } = this.props
 
-    return <InvertedButton style={[styles.default, style]} textStyle={[styles.text, textStyle]} {...props} />
+    return (
+      <View style={[styles.default, style]}>
```
Comment (l2succes) [inline src/lib/Components/Bidding/Components/Button.tsx]: @yuki24 how come we're not using a styled component here?

Comment (yuki24) [reply src/lib/Components/Bidding/Components/Button.tsx]: We can totally use it here! will update it.

File: src/lib/Components/Bidding/Components/Button.tsx [hunk 2]
```diff
@@ -21,7 +27,7 @@ export class BidGhostButton extends React.Component<ButtonProps> {
 
 const styles = StyleSheet.create({
   default: {
-    height: 46,
+    height: 50,
   },
```
Comment (yuki24) [inline src/lib/Components/Bidding/Components/Button.tsx]: We can't get rid of this non-styled CSS definition entirely due to the fact that the `<Button>` component is not built based on the styled-system.

**Comment by yuki24** [comment]
This PR is ready for review.

File: src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx [hunk 3]
```diff
@@ -50,6 +51,16 @@ describe("when pressing bid button", () => {
...
   })
 
+  it("shows a spinner", () => {
+    const component = renderer.create(<ConfirmBid {...initialProps} />)
+    component.root.instance.setState({ conditionsOfSaleChecked: true })
+    relay.commitMutation = jest.fn()
+
+    component.root.findByType(Button).instance.props.onPress()
+
+    expect(component.root.findAllByType(Spinner).length).toEqual(1)
```
Comment (ashfurrow) [inline src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx]: Nice!

**Comment by sweir27** [comment]
Looks awesome! Everyone else gave a review already so I'm going to go ahead and merge...

**Pull request merged**